### ISSUE TITLE
pvpython

### ISSUE THREAD

**Issue by ThomasWollmann** [conda-forge/paraview-feedstock#4]
When executing "from vtk import *" I receive "ImportError: No module named vtkCommonCorePython". Therefore you can't do anything. Why is this Module not available?

**Comment by jschueller** [comment]
The python install of paraview is not working properly so we have to hack a bit, see the build.sh.
It requires a paraview.vtk module but the latter is not working so I moved the vtk module inside:
https://github.com/conda-forge/paraview-feedstock/blob/master/recipe/build.sh#L54

**Comment by ThomasWollmann** [comment]
Hmm still seems that the .so file for vtkCommonCorePython is not loaded properly. How can we fix that?

**Comment by jschueller** [comment]
What ? Where ?

**Comment by rpep** [comment]
I'm also having this problem. On a fresh install of Paraview through Conda-forge, I get the error:

```
>>> from paraview.simple import *
Traceback (most recent call last):
  File "/home/ryan/miniconda3/lib/site-packages/vtk/vtkCommonCore.py", line 5, in <module>
    from .vtkCommonCorePython import *
ModuleNotFoundError: No module named 'vtk.vtkCommonCorePython'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "scripts/square-generate-png-ground-state.py", line 10, in <module>
    from render import *
  File "/media/ryan/bigdata/data-paper-modified-docker/scripts/render.py", line 2, in <module>
    from paraview.simple import *
  File "/home/ryan/miniconda3/lib/python3.6/site-packages/paraview/simple.py", line 43, in <module>
    from paraview import servermanager
  File "/home/ryan/miniconda3/lib/python3.6/site-packages/paraview/servermanager.py", line 53, in <module>
    from paraview import vtk
  File "/home/ryan/miniconda3/lib/python3.6/site-packages/paraview/vtk/__init__.py", line 174, in <module>
    from .util.misc import calldata_type
  File "/home/ryan/miniconda3/lib/python3.6/site-packages/paraview/vtk/util/misc.py", line 4, in <module>
    import sys, os, vtk
  File "/home/ryan/miniconda3/lib/site-packages/vtk/__init__.py", line 41, in <module>
    from .vtkCommonCore import *
  File "/home/ryan/miniconda3/lib/site-packages/vtk/vtkCommonCore.py", line 9, in <module>
    from vtkCommonCorePython import *
ModuleNotFoundError: No module named 'vtkCommonCorePython'
```

I then found though that VTK wasn't even installed as a dependency of Paraview in the channel, so something is going wrong here.

**Comment by jschueller** [comment]
Should be fixed by 5.5.2
